styleguide/source/_patterns/01-molecules/01-layouts/01-layout-two-up/layout-two-up.twig

```diff
@@ -1,10 +1,10 @@
 <div class="layout layout-example layout_two_up">
     <div class="layout_two_up-primary">
-        <div class="layout_container">2-up</div>
-        <div class="layout_container">2-up</div>
+        <div class="layout_container">2-up<br />1st</div>
```

Comment:
I disagree on this. <br/> is legitimate HTML5 markup, used to break up content for presentational purposes. Since this is pure placeholder content, it seems like an appropriate use case for <br> as opposed to using styles to adjust the presentation or wrapping things in <divs> arbitrarily.